src/app/challenges/submission/page.tsx

```diff
@@ -6,7 +6,7 @@ import TallyForm from "@/components/TallyForm";
 const SubmissionPage = () => {
   return (
     <TallyForm
-      tallyUrl="https://tally.so/r/wLYDW1"
+      tallyUrl="https://tally.so/r/mRBp4d"
```

Comment:
I think this is the wrong URL? I'm getting the `Register for the Scroll Articles Bounty Contest: Write and Earn!` form

Reply:
![CleanShot 2024-11-12 at 22 06 51@2x](https://github.com/user-attachments/assets/1ed86367-29fd-4db4-ba50-9b4f545eb305)
could you re-confirm it with Emily? @isabellewei